css-view-transitions-1/Overview.bs

```diff
@@ -47,6 +47,9 @@ spec:css-display-4; type: dfn; text:invisible;
 urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 	text: NavigateEvent
 	text: signal; for: NavigateEvent; url: #ref-for-dom-navigateevent-signal①
+urlPrefix: https://html.spec.whatwg.org/multipage/; spec: html; type: dfn;
+	text: restore persisted state; url: #restore-persisted-state
+	text: latest entry; url: #latest-entry
```

Comment:
Instead of poking into HTML's externals, try to get https://github.com/whatwg/html/pull/9996 to export the appropriate parts.

(I'm a bit scared of exporting "latest entry" though, since it's a pretty low-level detail... document's node navigable's active session history entry would be a more public API. Maybe it's the same thing in this case?)